src/interfaces/records/handlers/records-write.ts

```diff
@@ -32,7 +31,7 @@ export class RecordsWriteHandler implements MethodHandler {
     // authentication & authorization
     try {
       await authenticate(message.authorization, this.didResolver);
-      await recordsWrite.authorize(tenant, this.messageStore);
+      await recordsWrite.authorize(tenant, this.storageController.MessageStore);
```

Comment:
ditto [src/interfaces/records/handlers/records-read.ts:56](https://github.com/TBD54566975/dwn-sdk-js/pull/338/files#diff-72cc0e5d5cf69521228f5e6253cb4d3be3b0888330231880addb51e836c24f6fR56) but as `authorizeRecordsWrite`

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/10146b17e43411da36e2e7a615a26c726441d9b7